website/docs/docs/cloud-integrations/downstream-exposures-tableau.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Configure exposures automatically"
```

Comment:
The way we're phrasing this might be a little confusing. Should it be something like:
```suggestion
title: "Configure automatic downstream exposures"
```